src/pages/api/termTime.ts

```diff
@@ -0,0 +1,37 @@
+import { NextApiResponse } from 'next';
+import { redirectToError, redirectTo } from './apiUtils/index';
+import { TERM_TIME_ATTRIBUTE } from '../../constants/index';
+import { isSessionValid } from './apiUtils/validator';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { NextApiRequestWithSession } from '../../interfaces';
+
+export interface TermTimeAttribute {
+    termTime: boolean;
```

Comment:
Does this want to be in the index of types? As it will be needed on the matching JSON API's.